README.md

```diff
@@ -218,6 +218,14 @@ Options:
 - `bitLength`: Number of id bits. (default: 128)
 - `multibase`: Account for multibase encoding. (default: true)
 
+### `generateMultibaseSeed()`
```

Comment:
Add example usage code, for these two new entries.

Reply:
Added 09c7750